doc/news/changes/minor/20180715Jean-PaulPelteret

```diff
@@ -0,0 +1,6 @@
+Changed: The Subscriptor class has been made thread-safe, and its behavior in
+debug and release modes are now aligned. Although this makes subscription and 
```

Comment:
I think it would make more sense to write "its behavior in debug and release modes are now identical."